client/wfnews-war/src/main/angular/src/app/components/report-of-fire/review-page/rof-review-page.component.ts

```diff
@@ -200,6 +203,28 @@ export class RoFReviewPage extends RoFPage implements AfterViewInit{
           iconAnchor: [ 14, 14 ]
       } )
     }).addTo(this.map)
+
+    // draw the arrow and lines between fire location and current location
+    let latlngs = Array();
```

Comment:
For the code in this block, there should be some defensive checks for nulls. Specifically on the reportOfFire and fireLocation stuff which will throw errors if there are undefined or null parts.

Reply:
thanks. Updated